src/components/ChatInput/ChatInput.js

```diff
@@ -16,6 +16,11 @@ const ChatInput = () => {
   const inputRef = useRef(null);
   const messageRef = useRef();
   const [disableButton, setDisableButton] = useState(true);
+  const [members, setMembers] = useState({});
+  const [filteredMembers, setFilteredMembers] = useState([]);
+  const [mentionIndex, setmentionIndex] = useState(-1);
+  const [startReading, setStartReading] = useState(false);
```

Comment:
why do we need this?

Reply:
_mentionIndex_ is used to store the index of selected user from members list. I think we should drop startReading and use showMembersList instead to show the list when user types '@' till ' ' is typed.

Reply:
Ohh just realized we can't remove startReading state, this is because if user hits backspace after selecting a member from list, since startReading will be true it will still show the filtered members.

Reply:
Got it, Thanks!